### ISSUE TITLE
Conflicting reports

### ISSUE THREAD

**Issue by Ais8Ooz8** [FairwindsOps/bif#12]
### What happened?

```
Input: alpine 3.17.1 

   BASE IMAGE   | LAST SCAN  |      CVE      | SEVERITY | CVSS |    FIXED IN     
----------------+------------+---------------+----------+------+-----------------
  alpine:3.17.1 | 2023-02-28 | CVE-2022-4450 | HIGH     | 7.50 | 3.18.3, 3.17.5  
                |            | CVE-2023-0215 | HIGH     | 7.50 | 3.18.3, 3.17.5  
                |            | CVE-2023-0216 | HIGH     | 7.50 | 3.18.3, 3.17.5  
                |            | CVE-2023-0217 | HIGH     | 7.50 | 3.18.3, 3.17.5  
                |            | CVE-2023-0286 | HIGH     | 7.40 | 3.18.3, 3.17.5  
                |            | CVE-2023-0401 | HIGH     | 7.50 | 3.18.3, 3.17.5  
                |            | CVE-2022-4203 | MEDIUM   | 0.00 | 3.18.3, 3.17.5  
                |            | CVE-2022-4304 | MEDIUM   | 5.90 | 3.18.3, 3.17.5  
```

```
Input: alpine 3.17.2 

   BASE IMAGE   | LAST SCAN  | CVE | SEVERITY | CVSS | FIXED IN  
----------------+------------+-----+----------+------+-----------
  alpine:3.17.2 | 2023-02-28 |     |          | 0.00 |           
```

### What did you expect to happen?

I expected the absence of contradictions and compliance with the publications on the official website.

https://www.alpinelinux.org/posts/

### How can we reproduce this?

```
bif find --image alpine:3.17.1
```
```
bif find --image alpine:3.17.2
```

### Version

Version:0.0.11 Commit:71958f17678bc1a7d41093c1c630ca49063db8d4

### Search

- [X] I did search for other open and closed issues before opening this.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by sudermanjr** [comment]
This definitely looks off. I'll have a look

**Comment by sudermanjr** [comment]
Seems like maybe I need to fix the scanning mechanism to update the scans. I fixed the 3.17.2 image for now.

**Comment by sudermanjr** [comment]
Fixed for these images. Other internal work is ongoing to prevent future occurences
